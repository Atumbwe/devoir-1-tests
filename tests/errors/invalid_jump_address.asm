jump 15
stack
