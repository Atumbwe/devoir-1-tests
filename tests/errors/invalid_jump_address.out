ERROR (jump): invalid jump address 15
[ ]
